doc/source/user_guide/advanced.rst

```diff
@@ -965,21 +965,28 @@ If you select a label *contained* within an interval, this will also select the
    df.loc[2.5]
    df.loc[[2.5, 3.5]]
 
-``Interval`` and ``IntervalIndex`` are used by ``cut`` and ``qcut``:
+:func:`cut` and :func:`qcut` both return a ``Categorical`` object, and the bins they
+create are stored as an ``IntervalIndex`` in its `.categories` attribute.
 
 .. ipython:: python
 
    c = pd.cut(range(4), bins=2)
    c
    c.categories
 
-Furthermore, ``IntervalIndex`` allows one to bin *other* data with these same
-bins, with ``NaN`` representing a missing value similar to other dtypes.
+:func:`cut` also accepts an ``IntervalIndex`` for its `bins` argument, which enables
```

Comment:
Double backticks for anything that is not a role here

Reply:
fixed.